src/tinypeel/tinypeel.py

```diff
@@ -416,28 +562,45 @@ def getArgs():
 
 def main():
     args = getArgs()
+
+    args.bfile = args.plink_file
+    args.genotypes = args.geno_file
+    args.phasefile = args.hap_file
+    args.seqfile = args.seq_file
+    args.pedigree = args.ped_file
+    args.startsnp = args.start_snp
+    args.stopsnp = args.stop_snp
+    args.writekey = args.out_id_order
+    args.onlykeyed = args.out_id_only
+    args.iothreads = args.n_io_thread
+    args.maxthreads = args.n_thread
+
```

Comment:
Please add a line  of `args.segfile = args.seg_file`.

Reply:
ok